packages/util/src/Percentage.ts

```diff
@@ -0,0 +1,48 @@
+import { BigIntMath } from './BigIntMath';
+
+/**
+ * Calculates the rounded percentages for each part from {@link total}.
+ * When total is omitted, it is assumed that the total is the sum of the parts.
+ *
+ * Dividing bigints is always rounded down, creating this problem:
+ * sum([...percentage(part)]) != percentage(sum([...parts]))
+ * This function will be distribute the difference to the highest percentage value.
+ *
+ * @param parts array bigint values
+ * @param total optional param to allow sum(parts) to be smaller than the total
+ * @returns array of integer percentages
+ */
+export const calcRoundedPercentages = (parts: bigint[], total: bigint = BigIntMath.sum(parts)): number[] => {
```

Comment:
Would be great to be consistent when representing percentage. This type represents percentage as floating point though.
```suggestion
export const calcRoundedPercentages = (parts: bigint[], total: bigint = BigIntMath.sum(parts)): Cardano.Percent[] => {
```

Reply:
Updated code to use floating point percentages in:
- [98115e8](https://github.com/input-output-hk/cardano-js-sdk/pull/735/commits/98115e844dfaac0fa1128b9c7b84acb6d9d53c90)
- [73358ab](https://github.com/input-output-hk/cardano-js-sdk/pull/735/commits/73358ab247f8b3c4ffe5dff72116e433f3503825)
- [4a1e70c](https://github.com/input-output-hk/cardano-js-sdk/pull/735/commits/4a1e70c437f0f710b0f098e7831f52bfb542c58c)
- [a2b9155](https://github.com/input-output-hk/cardano-js-sdk/pull/735/commits/a2b9155d0207557807c732290fc428826f1a7bd6)